src/main/java/seedu/address/model/Model.java

```diff
@@ -193,51 +193,70 @@
     void appendCommandHistoryEntry(String commandText);
 
     /**
-     * Returns true if a issue with the same identity as {@code issue} exists in the address book.
+     * Returns true if a issue with the same identity as {@code issue} exists in SunRez.
+     *
+     * @param issue to be checked
+     * @return {@code True} if issue is in SunRez
+     * @throws NullPointerException if {@code issue} is null.
      */
     boolean hasIssue(Issue issue);
 
     /**
      * Deletes the given issue.
      * The issue must exist in SunRez.
+     *
+     * @param target to be deleted
```

Comment:
Missing punctuation and captialisation. Param description should be separated from param name.